packages/frontend/src/components/FlowRow/DemoFlowModal.tsx

```diff
@@ -0,0 +1,72 @@
+import { ReactElement, useState } from 'react'
+import { BiBulb } from 'react-icons/bi'
+import {
+  Flex,
+  Image,
+  Modal,
+  ModalContent,
+  ModalFooter,
+  ModalOverlay,
+  Text,
+} from '@chakra-ui/react'
+import { Badge, BadgeLeftIcon, Button } from '@opengovsg/design-system-react'
+import demoModalImg from 'assets/demo-modal.png'
+
+import DemoVideoModalContent from './DemoVideoModalContent'
+
+interface DemoFlowModalProps {
+  onClose: () => void
+}
+
+export default function DemoFlowModal(props: DemoFlowModalProps): ReactElement {
+  const { onClose } = props
+  const [showVideoModal, setShowVideoModal] = useState(false)
+
+  return (
+    <Modal
+      isOpen={true}
+      onClose={onClose}
+      size={showVideoModal ? '5xl' : '3xl'}
+      motionPreset="none"
+      closeOnEsc={false}
+    >
+      <ModalOverlay bg="base.canvas.overlay" />
+      <ModalContent my={12}>
+        {showVideoModal ? (
+          // TODO (mal): to update this when there are more demo links in the future
+          <DemoVideoModalContent
+            src="https://demo.arcade.software/FzpL1zCmibw0oXR6HUJi?embed&show_copy_link=true"
+            title={`Forms -&gt; Postman (GGWP)`}
```

Comment:
remove unnecessary

Reply:
actly can we just leave the title for `iframe` for accessibility purposes?